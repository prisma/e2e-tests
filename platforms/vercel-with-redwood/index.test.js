const { request, gql } = require('graphql-request')

const query = gql`
  query {
    prismaVersion
    users {
      id
      email
      name
    }
  }
`

test('should query graphql users', async () => {
  const data = await request(
    // 'https://e2e-vercel-with-redwood.vercel.app/api/graphql',
    'http://localhost:8911/graphql',
    query
  )
  expect(data).toMatchSnapshot()
})
