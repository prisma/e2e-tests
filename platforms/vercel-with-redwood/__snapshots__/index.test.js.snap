// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should query graphql users 1`] = `
Object {
  "users": Array [],
}
`;
